core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -233,7 +233,7 @@ public static function decodeEntities($strString, $strQuoteStyle=ENT_COMPAT, $st
 
 		if ($strCharset === null)
 		{
-			$strCharset = Config::get('characterSet');
+			$strCharset = $GLOBALS['TL_CONFIG']['characterSet'] ?? 'UTF-8';
```

Comment:
No `System::getContainer()->getParameter('kernel.charset')` here?

Reply:
`StringUtil` is so widely used, that `System::getContainer()` would require us to update many many unit-test. And as `$GLOBALS['TL_CONFIG']['characterSet']` is practically always `UTF-8` I thought this is the better compromise.

Reply:
IMHO we could also just set it to `UTF-8` here.

Reply:
```suggestion
			$strCharset = 'UTF-8';
```

Reply:
Done in 1be8e58f2609f30a32cddd2ea5dffdde35cd1cc6